valerymo/integreatly-operator
# Mgdapi 2712 - Move SSO dashboards & alerts to the OO namespace

# Issue link

https://issues.redhat.com/browse/MGDAPI-2712

# What

Prometheus and Grafana can not see outside of their namespace. The SSO operator creates dashboards and prometheus rules in its own namespace. This work exports the required CR's from the SSO namespaces into the correct namespace. 

# Verification steps




## Verification 1
Checking the Grafana dashborads. Metrics are missing from this PR, the dashboard will be missing information. As the dashboard allows to select which instance of SSO to display information on, only one dashboard is copied over.

- install this branch
- navigate to the Grafana instances `oc get routes -n redhat-rhoam-observability --no-headers | grep grafana | awk '{print $2}' | xargs -I {} xdg-open https://{}`
- Expected: dashboard called `keycloak-metrics` exists.

## Verification 2
Check alerts for both SSO instances are present. Also with the metrics missing some of these alerts are not working.

- install this branch
- navigate to the Prometheus instances `oc get routes -n redhat-rhoam-observability --no-headers | grep prometheus | awk '{print $2}' | xargs -I {} xdg-open https://{}`
- Go to the alerts tab.
- Expected: The follow files should be present
  - redhat-rhoam-observability-rhsso.yaml
  - redhat-rhoam-observability-rhssouser.yaml
